cryodrgn/commands/filter.py

```diff
@@ -234,6 +241,8 @@ def main(args: argparse.Namespace) -> None:
         if save_option == "yes":
             if args.force:
                 filename = "indices"
+                if args.sel_dir:
+                    filename = os.path.join(args.sel_dir, filename)
```

Comment:
if `args.sel_dir` is given we should prepend it to the output filename whether or not `args.force` is given